keps/sig-auth/1513-certificate-signing-request/README.md

```diff
@@ -323,6 +328,11 @@ type CertificateSigningRequestSpec struct {
   // You can select on this field using `.spec.signerName`.
   SignerName string
 
+  // durationHint is a hint to the signer in regards to when the issued certificate should expire.
+  // The signer may or may not honor this field.  The well-known kubernetes signers will honor this field
+  // as long as the requested duration is not later than the maximum duration they will honor.
+  DurationHint *metav1.Duration
```

Comment:
per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units, include unit in the field name and make it an int32 (we don't use time.Duration / metav1.Duration in REST APIs since it requires go parsing to decode)